Mesh_3/include/CGAL/Mesh_criteria_3.h

```diff
@@ -73,7 +73,10 @@ class Mesh_criteria_3_impl
     :edge_criteria_(parameters::choose_parameter(parameters::get_parameter(np, internal_np::edge_size_param),
                                                  parameters::choose_parameter(parameters::get_parameter_reference(np, internal_np::edge_sizing_field_param),
                                                                               parameters::choose_parameter(parameters::get_parameter(np, internal_np::sizing_field_param), FT(DBL_MAX)))),
-                    parameters::choose_parameter(parameters::get_parameter(np, internal_np::edge_min_size_param), FT(0))),
+                    parameters::choose_parameter(parameters::get_parameter(np, internal_np::edge_min_size_param), FT(0)),
+                    parameters::choose_parameter(parameters::get_parameter(np, internal_np::edge_distance_param),
+                                                 parameters::choose_parameter(parameters::get_parameter_reference(np, internal_np::edge_distance_sizing_field_param),
+                                                                              parameters::choose_parameter(parameters::get_parameter(np, internal_np::sizing_field_param), FT(DBL_MAX))))),
```

Comment:
```suggestion
                    parameters::choose_parameter(parameters::get_parameter(np, internal_np::edge_min_size_param), FT(0)),
                    parameters::choose_parameter(parameters::get_parameter(np, internal_np::edge_distance_param), FT(0))),
```

check the nb of parenthesis before committing this suggestion...